tensorflow/c/c_api.h

```diff
@@ -1577,6 +1578,70 @@ TF_CAPI_EXPORT extern void TF_RegisterLogListener(
 TF_CAPI_EXPORT extern void TF_RegisterFilesystemPlugin(
     const char* plugin_filename, TF_Status* status);
 
+// Apis that are correponding to python c api. --------------------
+
+// Add control input to `op`.
+TF_CAPI_EXPORT extern void TF_AddOperationControlInput(TF_Graph* graph, TF_Operation* op, TF_Operation* input);
+
+// Changes an attr value in the node_def Protocol Buffer and sets a status upon
+// completion.
+TF_CAPI_EXPORT extern void TF_SetAttr(TF_Graph* graph, TF_Operation* op,
+                                       const char* attr_name,
+                                       TF_Buffer* attr_value_proto,
+                                       TF_Status* status);
+
+// Clears the attr in the node_def Protocol Buffer and sets a status upon
+// completion.
+TF_CAPI_EXPORT extern void TF_ClearAttr(TF_Graph* graph, TF_Operation* op,
+                                         const char* attr_name,
+                                         TF_Status* status);
+
+// Sets the experimental_type` field in the node_def Protocol Buffer.
+TF_CAPI_EXPORT extern void TF_SetFullType(TF_Graph* graph, TF_Operation* op,
+                                           const tensorflow::FullTypeDef& full_type);
+
+TF_CAPI_EXPORT extern void TF_SetRequestedDevice(TF_Graph* graph,
+                                                  TF_Operation* op,
+                                                  const char* device);
+
+TF_CAPI_EXPORT extern void TF_RemoveAllControlInputs(TF_Graph* graph, TF_Operation* op);
```

Comment:
Please add docstrings for both these APIs

Reply:
Thank you for your reminder. I've added docstrings to them.